addOns/oast/CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Changed
+- Updated service URLs for Boast and Interactsh in help content.
```

Comment:
- e.g. "Link to repositories/documentation instead of service URLs in the help content." would be more clear on what was done IMO. 
- Should be BOAST